File: Week-04/1. Primitive/Solutions/ROHITKUMAR(GCS1940022)/solution2.py
#code
t=int(input())
for i in range(t):
    n=int(input())
    arr=list(map(int,input().split()))
    sum1=0
    sum2=0
    for j in range(n//2):
        sum1+=arr[j]
    for j in range(n//2,n):
        sum2+=arr[j]
    print(sum1*sum2)
    

File: Week-04/1. Primitive/Solutions/MasoomMittal (GCS-1940066)/Solution2.cpp
#include <iostream>
using namespace std;

int main() {
	//code
	int t;
	cin>>t;
	for(int i=0;i<t;i++)
	{
	    int n,sum1=0,sum2=0;
	    cin>>n;
	    int a[n];
	    for(int i=0;i<n;i++)
	    {
	        cin>>a[i];
	    }
	    int c=n/2;
	    for(int i=0;i<c;i++)
	    {
	        sum1+=a[i];
	    }
	    for(int i=c;i<n;i++)
	    {
	        sum2+=a[i];
	    }
	    cout<<sum1*sum2<<endl;
	}
	return 0;
}
